File: Proyecto02/T_Registros+CoreDataProperties.m
//
//  T_Registros+CoreDataProperties.m
//  Proyecto02
//
//  Created by MacBook Pro on 17/11/25.
//
//

#import "T_Registros+CoreDataProperties.h"

@implementation T_Registros (CoreDataProperties)

+ (NSFetchRequest<T_Registros *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"T_Registros"];
}

@dynamic habitLog;
@dynamic amount;
@dynamic date;

@end

File: Proyecto02/T_Registros+CoreDataProperties.swift
//
//  T_Registros+CoreDataProperties.swift
//  Proyecto02
//
//  Created by MacBook Pro on 17/11/25.
//
//

import Foundation
import CoreData


extension T_Registros {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<T_Registros> {
        return NSFetchRequest<T_Registros>(entityName: "T_Registros")
    }

    @NSManaged public var habitLog: String?
    @NSManaged public var amount: Double
    @NSManaged public var date: Date?

}

extension T_Registros : Identifiable {

}
